### DISCUSSION TITLE
[4.0] Write user action log on plugin reordering

### DISCUSSION THREAD

**Discussion by LeonardoGalardi** [joomla/joomla-cms#26719] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

When I change the ordering of plugins it doesn't write the user log

### Describe the solution you'd like

log the action

**Comment by adj9** [comment]
I did some tests with **Joomla 3.9.12**. you can move the deactivated plugs in the direction of the arrow above (second icon). 
The actions in "User Actions Log" are not shown.
Active plungins do not move. Equivalent behavior for the modules.
Instead, for the articles it is possible to move the active ones.

All this following the activation of the function in question.

I advise you not to record these actions as being minimal (or to be able to make a choice).
